pkg/util/annotations/helpers.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2024 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package annotations
```

Comment:
How about moving these files under `pkg/util/vmopv1` instead? That's kind of mostly where we've been dumping util-y things that imports vmopv1

Reply:
Started making this change before I realized.  This *isn't* related to vmopv1 (even though it imports it).  This is a generic function which only operates on the object's metadata and checks whether an annotation (string) is present.  The schema is only imported for the strings :( 

I think a "pkg/util/vmopv1/annotations.HasPaused" package reads like it is specifically written for a schema (v1a3 in this case).  That's why I think it belongs in pkg/util subdir.  Let me know your thoughts.

Reply:
But it is using an annotation defined in our vmopv1 api. Outside of conversion everything just cares about the stoarage version.

Reply:
Resolving this for now.  We can move the file later if needed.